src/main/java/appeng/block/misc/BlockSkyCompass.java

```diff
@@ -164,4 +186,31 @@ public void addCollidingBlockToList( final World w, final BlockPos pos, final Ax
 	{
 
 	}
+
+	@Override
+	public EnumBlockRenderType getRenderType( IBlockState state )
+	{
+		return EnumBlockRenderType.ENTITYBLOCK_ANIMATED;
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public TileEntitySpecialRenderer<? extends AEBaseTile> getTESR()
+	{
+		return new SkyCompassTESR();
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public ItemMeshDefinition getItemMeshDefinition()
+	{
+		return is -> new ModelResourceLocation( "appliedenergistics2:tile.BlockSkyCompass", "normal" );
```

Comment:
I made that assumption based on the AE2 gradle script setting source/target to Java 8 explicitly. I hope that was correct to assume. I've used Lambdas/Streams in other PRs already. (And I would second using Java 8 Optional, although I'd do  an automated replacement in a separate PR)